worlds/outer_wilds/shared_static_logic/connections.jsonc

```diff
@@ -32,9 +32,11 @@
     // This connection is tricky because you need to reach the forge controls in Hanging City first,
     // and then get back to whichever warp pad connects to Hanging City Ceiling _in the same loop_.
     // I believe that requires either the Spaceship or a "warp path" from the Northern Glacier pad to
-    // the Hanging City Ceiling pad. The latter is hard to write logic for, so we require the ship.
-    // And once the ship is required, { "region": "Brittle Hollow" } would be redundant.
-    { "from": "Hanging City Ceiling", "to": "Black Hole Forge", "requires": [ { "item": "Launch Codes" } ] },
+    // the Hanging City Ceiling pad. The warp path connection is conditionally added elsewhere if it exists.
+    { "from": "Hanging City Ceiling", "to": "Forge via Ship", "requires": [ { "item": "Launch Codes" } ] },
+    
+    { "from": "Forge via Ship", "to": "Black Hole Forge", "requires": [] },
+    { "from": "Forge via Warps", "to": "Black Hole Forge", "requires": [] },
```

Comment:
This is probably an annoying request to make now, but since these are player-facing strings we're gonna be showing in the in-game tracker: I think the regions should be called "Forge via Ship & Warps" and "Forge via Warps Only". The current 1st name sounds too much like ship _only_, as if you'd land the ship directly on the ceiling or on the forge itself at some weird angle.

Also applies to the other PR.